microsoft-edge/extensions-chromium/developer-guide/declare-permissions.md

```diff
@@ -38,13 +38,36 @@ The following code outlines how to declare permissions in the manifest file:
 ]
 ```
 
+<!-- per upstream doc:
+todo: differs in v2 vs v3?
+-->
```

Comment:
upstream URL: tbd
Added 2nd version of this "permissions part..." below the first.  Which is correct/current?  
rendered page section: tbd